Maddori.Apple/Maddori.Apple/Screen/MyReflection/FeedbackToMeDetail/FeedbackToMeDetailViewController.swift

```diff
@@ -0,0 +1,187 @@
+            $0.height.equalTo(myReflectionScrollView.snp.height).offset(50)
+        }
+        
+        myReflectionContentView.addSubview(keywordTitleLabel)
+        keywordTitleLabel.snp.makeConstraints {
+            $0.top.equalToSuperview().inset(SizeLiteral.topPadding)
+            $0.leading.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+        }
+        
+        myReflectionContentView.addSubview(feedbackTypeLabel)
+        feedbackTypeLabel.snp.makeConstraints {
+            $0.top.equalTo(keywordTitleLabel.snp.bottom).offset(SizeLiteral.topComponentPadding)
+            $0.leading.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+        }
+        
+        myReflectionContentView.addSubview(feedbackTypeText)
+        feedbackTypeText.snp.makeConstraints {
+            $0.top.equalTo(feedbackTypeLabel.snp.bottom).offset(SizeLiteral.labelComponentPadding)
+            $0.leading.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+        }
+        
+        myReflectionContentView.addSubview(feedbackFromLabel)
+        feedbackFromLabel.snp.makeConstraints {
+            $0.top.equalTo(feedbackTypeText.snp.bottom).offset(SizeLiteral.componentIntervalPadding)
+            $0.leading.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+        }
+        
+        myReflectionContentView.addSubview(feedbackFromText)
+        feedbackFromText.snp.makeConstraints {
+            $0.top.equalTo(feedbackFromLabel.snp.bottom).offset(SizeLiteral.labelComponentPadding)
+            $0.leading.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+        }
+        
+        myReflectionContentView.addSubview(feedbackContentLabel)
+        feedbackContentLabel.snp.makeConstraints {
+            $0.top.equalTo(feedbackFromText.snp.bottom).offset(SizeLiteral.componentIntervalPadding)
+            $0.leading.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+        }
+        
+        myReflectionContentView.addSubview(feedbackContentText)
+        feedbackContentText.snp.makeConstraints {
+            $0.top.equalTo(feedbackContentLabel.snp.bottom).offset(SizeLiteral.labelComponentPadding)
+            $0.leading.trailing.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+        }
+        
+        myReflectionContentView.addSubview(feedbackStartLabel)
+        feedbackStartLabel.snp.makeConstraints {
+            $0.top.equalTo(feedbackContentText.snp.bottom).offset(SizeLiteral.componentIntervalPadding)
+            $0.leading.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+        }
+        
+        myReflectionContentView.addSubview(feedbackStartText)
+        feedbackStartText.snp.makeConstraints {
+            $0.top.equalTo(feedbackStartLabel.snp.bottom).offset(SizeLiteral.labelComponentPadding)
+            $0.leading.trailing.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+        }
```

Comment:
p1;

```suggestion
        feedbackStartText.snp.makeConstraints {
            $0.top.equalTo(feedbackStartLabel.snp.bottom).offset(SizeLiteral.labelComponentPadding)
            $0.leading.trailing.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
            $0.bottom.equalToSuperview()
        }
```
아래까지 superView의 크기로 고정하게 된다면 예쁘게 동작하는걸 볼 수 있습니다 ^________^

https://user-images.githubusercontent.com/78677571/201793115-ae0ff2ef-4126-49aa-af25-59df2cfd397e.mp4

Reply:
[6175961](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/98/commits/61759614eac1aa77e23f2cde5b3f968214c4aa3c)
와우 완전 깔끔하게 되네요,, 밑에 inset 20만 추가해두었습니다 ! 감사합니당